metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_classes.py

```diff
@@ -41,6 +41,7 @@ class DatasetKey(ContainerKey):
 class Workspace:
     id: str
     name: str
+    type: str
```

Comment:
is this also being used as the subtype? if so, we add that as a comment to make it clear